README.md

```diff
@@ -45,3 +46,4 @@
 | [Victoria Gonda](https://twitter.com/TTGonda) | Chicago, IL | [victoriagonda.com/talks](http://www.victoriagonda.com/talks)
 | [Yash Prabhu](https://twitter.com/yashvprabhu) | Philadelphia, PA | [yprabhu.com/p/speaking.html](http://www.yprabhu.com/p/speaking.html) |
 | [Zarah Dominguez](https://twitter.com/zarahjutz) | Sydney, Australia | [zarah.dev/about](https://zarah.dev/about) |
+
```

Comment:
Hi, I don't see you in the private group. Would you mind DMing me there JIC your name there might be different?